tests/snapshots/test_extended_sampling_glob_multi_rule.json

```diff
@@ -17,6 +17,7 @@
       "test": "tag"
     },
     "metrics": {
+      "_dd.limit_psr": 1.0,
```

Comment:
Previously `_dd.limit_psr` was ONLY set if `DatadogSampler.limiter.rate_limit != DEFAULT_SAMPLING_RATE_LIMIT`. This restriction does not make sense. If the rate limiter is used to make a sampling decision then `_dd.limit_psr` should be set!

Reply:
Good point!

Reply:
Sent docs on slack